index.html

```diff
@@ -25,19 +28,23 @@
     <header class="profile">
       <nav class="profile-menu">
```

Comment:
- [ ] Like the previous reviewer said the hamburger button and the x button is missing in the navbar kindly add those buttons.
- [ ] For mobile devices, the navbar has to overlay the whole size of the screen when the hamburger button is press.
![Captura de pantalla_2021-10-04_18-04-26](https://user-images.githubusercontent.com/53111464/135936300-29e9f475-80ee-4662-a957-e726d40aa4af.png)
- As you may see at the top there is a space that hasn't be cover.

- [ ] Kindly stick to the Figma design. Some margins, background images, and icons are missing.
**Figma**
![Captura de pantalla_2021-10-04_18-10-04](https://user-images.githubusercontent.com/53111464/135937293-7491d8f8-62ed-46dc-88d6-783057e8f0e2.png)
- [ ] `When the user clicks (or taps) the close (X) button, the mobile menu disappear.`
- [ ] `When the user clicks (or taps) any option of the mobile menu, the mobile menu disappear.`